raisinbread/RB_files/RB_Config.sql

```diff
@@ -91,9 +91,9 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (105,104,'Ymd');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (106,105,'Y-m-d H:i:s');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (107,106,'/data/issue_tracker/');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (108,107,'');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (113,112,'warning');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (114,113,'warning');
```

Comment:
isn't "error" a higher error level than "warning" (ie. it will show less errors?)

Reply:
you are right!